pathfinder/paizo/campaign_setting/inner_sea_magic/ism_abilities_class.lst

```diff
@@ -227,17 +269,49 @@ Black Blood Spray	KEY:Oracle ~ Black Blood Spray	CATEGORY:Special Ability	TYPE:O
+Bloodline Tattoo			KEY:Bloodline Tattoo ~ Tattooed Sorcerer						CATEGORY:Special Ability	TYPE:SorcererClassFeatures.SpecialQuality.Extraordinary																						DESCISPI:YES	DESC:Whenever a tattooed sorcerer gains a bloodline spell, a new tattoo manifests on her body to represent this spell. Her bloodline spells are always enhanced by her Varisian Tattoo feat, even if they don't match the school to which her Varisian Tattoo belongs.																																																																																																																																																																													SOURCEPAGE:p.40
+Create Spell Tattoo		KEY:Create Spell Tattoo ~ Tattooed Sorcerer					CATEGORY:Special Ability	TYPE:SorcererClassFeatures.SpecialQuality.Supernatural.SorcererBloodlineFeatLvl7	DEFINE:TattooedSorcererCreateSpellTattooTimes|0	BONUS:VAR|TattooedSorcererCreateSpellTattooTimes|1				DESC:A tattooed sorcerer can create a spell tattoo %1 times per day with a single touch as a standard action. The recipient of the spell tattoo must be willing to receive the spell tattoo. If she gives the spell tattoo to herself, it does not count against the regular limit of magic tattoos she can have. The spell tattoo must be of a spell that she knows that has no material or focus component. She can maintain one spell tattoo created by this ability at a time---if she uses this ability again, the previous spell tattoo she created fades away. Spell tattoos she creates with Inscribe Magic Tattoo do not count against this limit. This ability replaces the bloodline feat gained at 7th level.|TattooedSorcererCreateSpellTattooTimes																																																													BONUS:VAR|TattooedSorcererCreateSpellTattooTimes|1|PRECLASS:1,Sorcerer=11	BONUS:VAR|TattooedSorcererCreateSpellTattooTimes|1|PRECLASS:1,Sorcerer=15	BONUS:VAR|BloodlineFeatProgression|-6	SOURCEPAGE:p.40
+Enhanced Varisian Tattoo	KEY:Enhanced Varisian Tattoo ~ Tattooed Sorcerer	NAMEISPI:YES	CATEGORY:Special Ability	TYPE:SorcererClassFeatures.SpecialQuality.Supernatural.SorcererBloodlinePowerLvl9																		DESCISPI:YES	DESC:The tattooed sorcerer can pick any one spell she knows for which she has a Varisian Tattoo feat. This spell must be one that lacks focus components and costly material components. She can now use that spell as a spell-like ability once per day. This spell-like ability is not enhanced by her Varisian Tattoo, but it functions at +2 caster levels above her sorcerer caster level. Whenever she gains a bloodline power at a later date, she may change this spell-like ability to another qualifying spell. This ability replaces the 9th-level bloodline power.																																																																																																																											SOURCEPAGE:p.41
 # Varisian Tattoo Abilities (for Tattooed Sorcerer without Spell Focus feat)
```

Comment:
This one was already in - didn't modify it. Will fix to standard form & resubmit.